### DISCUSSION TITLE
Log setting: Show plot file paths and names for eligible plots

### DISCUSSION THREAD

**Discussion by Gregwolf3030** [Chia-Network/chia-blockchain#9779] [Ideas]
When monitoring my harvesters and there is an issue with a drive. The plots will respond slowly to lookups and produce the following type of log entry:

`2021-12-30T18:08:06.497 harvester chia.harvester.harvester: WARNING  Looking up qualities on M:\Portable_Plots\plot-k32-2021-07-10-04-00-760fd789eac2a3c55d8247a187d896e0a096dd6833a27fd7134383b9600c99fc.plot took: 5.569806814193726. This should be below 5 seconds to minimize risk of losing rewards.`

This log entry will be located next to the "# plots were eligible" entries and list the path and name of each plot file that was eligible allowing you to locate and troubleshoot those files.

My request is that there be a setting in the config file allowing you to turn on an entry similar to this that will list out the paths and file names of the eligible plots along with their respective lookup times. So this information is always shown, not just when the lookup times drop below a threshold like it is now. This would allow for more detailed troubleshooting and assist in fine tuning the farm.

**Comment by Jacek-ghub** [comment]
Great request.  I would also like to see another parameter, that would specify the borderline time.  This way, we could test the setup not just for 5 secs, but also for 2 or 3 seconds, ..., if we see some odd behaviors with lookup times.

Those both changes would really help to isolate "slow" or rather under-performing drives / controllers.

**Comment by pegleg81** [comment]
I second the request.  I have spent a lot of time trying to diagnose the long lookup times barking up the wrong tree (USB hubs, USB extension cables, Win power mgmt, Win indexing, virus scan, bios settings, windows updates, task scheduler, LSI HBA card firmware update, overloading Ivy Bridge pc, internet speed, home network load) and finally diagnosed it to a particular HDD that actually appears ok in SMART reports.

**Comment by Yamanipanuchi** [comment]
I would like throw my name in the hate for this feature also. I would seem that is lookup times were slow then naturally would would want to find where. There is little to no information to go on to narrow this down. Showing at least the eligible plots would allow you to trace it down threw process or elimination.
